src/backend/TrafficCourts/Staff.Service/Services/EmailHistoryService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MassTransit;
+using TrafficCourts.Common.Features.FilePersistence;
+using TrafficCourts.Common.OpenAPIs.OracleDataApi.v1_0;
+
+namespace TrafficCourts.Staff.Service.Services;
+
+/// <summary>
+/// Summary description for EmailHistoryService
+/// </summary>
+public class EmailHistoryService : IEmailHistoryService
+{
+    private readonly IOracleDataApiClient _oracleDataApi;
+    private readonly ILogger<EmailHistoryService> _logger;
+    private readonly IBus _bus;
+    private readonly IFilePersistenceService _filePersistenceService;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+
+
+    public EmailHistoryService(
+        IOracleDataApiClient oracleDataApi,
+        IBus bus,
```

Comment:
no using the bus in this class

Reply:
removed the bus, renamed FileHistoryRecord to SaveFileHistoryRecord which reflects the action happening